action.yml

```
@@ -5,22 +5,28 @@ inputs:
     description: 'Arguments passed to the builder'
     required: true
     default: '--help'
-  version:
-    description: 'The version of the builder'
-    required: false
-    default: 'dev'
 runs:
   using: "composite"
   steps: 
+    - shell: bash
+      id: version
+      run: |
+        input=$(echo "${{ github.action_path }}" | cut -d"/" -f8 )
+        if [[ "${input}" == "master" ]] || [[ -z "${input}" ]]; then
+          input="dev"
```

```suggestion
          input="latest"
```

Should we default to `latest` instead of `dev` for the container image?